service/src/main/java/bio/terra/workspace/common/utils/FlightUtils.java

```diff
@@ -281,10 +280,7 @@ public static boolean flightComplete(FlightState flightState) {
         "Testing flight {} completion; state is {}",
         flightState.getFlightId(),
         flightState.getFlightStatus());
-    // TODO: replace with call to flightState.isActive after TCL/Stairway is updated.
-    return (flightState.getFlightStatus() == FlightStatus.ERROR
-        || flightState.getFlightStatus() == FlightStatus.FATAL
-        || flightState.getFlightStatus() == FlightStatus.SUCCESS);
+    return !flightState.isActive();
```

Comment:
Implementation:

https://github.com/DataBiosphere/stairway/blob/62a5cbd35cc82787f7a43da18b0479c60d701283/stairway/src/main/java/bio/terra/stairway/FlightState.java#L108

The FlightUtils unit test I wrote in my previous PR does exercise calling this method.